lib/util.ts

```diff
@@ -278,3 +278,20 @@ export function delay(ms) {
 export function isPromise(obj) {
   return obj && obj.finally && (typeof obj.finally === 'function');
 }
+
+// browser only
+export function getUrlParts(url) {
+  const a = document.createElement('a');
```

Comment:
why not use the URL object? (which does work outside the browser): https://developer.mozilla.org/en-US/docs/Web/API/URL

Reply:
I was, then reported by the linter for ie11 compat issue.

Reply:
OK. (I love that linter BTW).

Reply:
Love the linter, hate IE11